ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -304,7 +301,7 @@ transitionImpl model cmd _ = eventAfter $ lockstep model cmd
 preconditionImpl :: Model Symbolic -> At CmdErr Symbolic -> Logic
 preconditionImpl Model{..} (At (CmdErr cmd mbErrors)) =
     compatibleWithError .&& case cmd of
-      GetPredecessor bids -> forall bids (`elem` bidsInModel)
+      GetBlockInfo bids -> forall bids (`elem` bidsInModel)
```

Comment:
Remove this precondition, `getBlockInfo` should just return `Nothing` for blocks not in the VolatileDB.